src/pages/certificates/index.tsx

```diff
@@ -15,59 +19,104 @@ import {
   CERTIFICATES_LIST_QUERY,
 } from './queries';
 import { CERTIFICATES_COLUMNS } from './table-config';
+import { Certificate, NewCertificateRequest } from './Certificate';
 import { Certificates } from '../../graphql/schema.types';
-import { AiFillSafetyCertificate } from 'react-icons/ai';
+import { DEFAULT_SORTERS } from '../../components/defaults';
+import { CertificatesListQuery } from '../../graphql/types';
+import { validateSync } from 'class-validator';
+import { plainToInstance } from 'class-transformer';
 
-export const CertificatesView: React.FC = () => {
-  return (
-    <GenericView
-      dtoClass={Certificate}
-      gqlQuery={CERTIFICATES_GET_QUERY}
-      editMutation={CERTIFICATES_EDIT_MUTATION}
-      createMutation={CERTIFICATES_CREATE_MUTATION}
-      deleteMutation={CERTIFICATES_DELETE_MUTATION}
-    />
+const BUTTON_TEXT = 'Generate Certificates';
+const DRAWER_TITLE = 'Generate New Certificate';
+const CERTIFICATES_API_URL = '/data/certificates/certificateChain';
```

Comment:
Seems something is off under the hood, when I run this and test it, seems the FE is making a http request to `ocpp/data/certificates/...`